Library/ViewModels/RewardCardViewModelTests.swift

```diff
@@ -90,6 +91,31 @@ final class RewardCardViewModelTests: TestCase {
     self.rewardTitleLabelText.assertValues(["Make a pledge without a reward"])
   }
 
+  func testTitleLabel_BackedNoReward() {
+    let reward = Reward.noReward
+
+    let project = Project.cosmicSurgery
+      |> Project.lens.state .~ .live
+      |> Project.lens.personalization.isBacking .~ true
+      |> Project.lens.personalization.backing .~ (
+        .template
+          |> Backing.lens.reward .~ reward
+          |> Backing.lens.rewardId .~ reward.id
+          |> Backing.lens.shippingAmount .~ 10
+          |> Backing.lens.amount .~ 700
+      )
+
+    self.vm.inputs.configureWith(
+      project: project,
+      rewardOrBacking: .left(reward)
```

Comment:
Just curious why wouldn't this be configured with the backing? 🤔

Reply:
Just helped me reason about the tests better I guess, I'd like to remove the `Either` here at some point, not sure we need it.